### DISCUSSION TITLE
The unknown extra diagnostic screen

### DISCUSSION THREAD

**Discussion by dryuemco** [microsoft/MixedRealityToolkit-Unity#10116] [General]
Hi all,

I just updated my app to 2.7 and I got this small screen. I don't know the name of it. How can I turn off or on this feature? What is its name? Is there any documentation for it? 

Thanks.
![20210727_143622_HoloLens](https://user-images.githubusercontent.com/6583833/127166487-5cf7e42c-5220-4520-a4cb-ddbadbdb9cca.jpg)

**Comment by keveleigh** [comment]
Not sure what this is! Are you using OpenXR or Windows XR Plugin?

**Reply by dryuemco** [reply]
I use OpenXR now. I wanted to take a look at experimental hand menu stuff and I got this.

**Reply by keveleigh** [reply]
Okay, looks like this is a setting in the OpenXR developer settings. If you look in your installed apps, you should see an OpenXR Developer Tools app (or you can install it from the store). In that app under developer settings, you should see this option:

![image](https://user-images.githubusercontent.com/3580640/127207240-c2167c2f-50e0-4de1-8380-294cb1769929.png)

which is what causes that panel to appear.

**Reply by dryuemco** [reply]
Ahh okay, thank you very much for that. It is in the HL2 menu and I successfully turned it off.
